Store/DataStoreModule.py

```diff
@@ -52,12 +52,28 @@ def __init__(self, db_username, db_password, db_host, db_port, db_name, db_type=
         # TEMP list of values for defaulted IDs, to be replaced by missing info lookup mechanism
         self.defaultUserId = 1  # DevUser
 
-        if db_type == 'sqlite':
+        # Instance attributes which are necessary for initialise method
+        self.db_name = db_name
+        self.db_type = db_type
+
+    def initialise(self):
```

Comment:
do we want to make the initialise() method an optional thing, just for writing the schema if required?
If so, we would want to leave the Base.metadata.bind = self.engine in the __init__ method

Reply:
I moved it into the init method and split Base into two declarative bases for each database in order to prevent conflicts of metadata of Postgres and SQLite:
https://github.com/debrief/python-import-library/pull/8/commits/7348d69c14a7a57c2d72548ac7f84680a8b8ecf8